app.js

```diff
@@ -1,12 +1,188 @@
+          <ul class="project-tools">
+              <ul class="mini__btn">
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[0]}</p>
+                  </li>
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[1]}</p>
+                  </li>
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[2]}</p>
+                  </li>
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[3]}</p>
+                  </li>
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[4]}</p>
+                  </li>
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[5]}</p>
+                  </li>
+              </ul>
+          </ul>
+          <div class="modal-btns">
+          <button>
+            <a href="${projects[cardNumber].seeLiveLink}">See Live <img src="image/tir.png" alt="See live"></i></a>
+          </button>
+          <button>
+            <a href="${projects[cardNumber].seeSourceLink}">See Source <img src="image/cartoon.png" alt="Github icon"></a>
+          </button>
+        </div>
+        </div>
+        </div>
+      </div>
+      </section>`;
+  document.addEventListener('click', (event) => {
+    if (event.target.id === 'close-popUp') {
+      popUp.style.display = 'none';
+    }
+  });
+}
+
+for (let i = 0; i < Btns.length; i += 1) {
+  Btns[i].addEventListener('click', () => {
+    popUpDesktop(i);
+  });
+}
+
+function popUpMobile(cardNumber) {
+  popUp.style.display = 'block';
+  projectWorkMobile.appendChild(popUp);
+  popUp.innerHTML = `
+        <div class="modal-content">
+        <div>
+        <div class="modal-header">
+        <h2>${projects[cardNumber].title}</h2>
+        <i class="fa fa-times" id="close-popUp" aria-hidden="true"></i>
+        </div>
+        <div class="card__tag card__tag__modal">
+            <div class="tag">
+              <p>Canopy</p>
+            </div>
+            <div class="tag__inactive">
+              <img src="image/Counter.png" alt="" />
+              <p>Back End Dev</p>
+            </div>
+            <div class="tag__inactive">
+              <img src="image/Counter.png" alt="" />
+              <p>2015</p>
+            </div>
+        </div>
+        <div class="modal-image">
+          <img src=${projects[cardNumber].src} alt="App Image">
+        </div>
+        <div class="modal-flex">
+        <div class="modal-description">
+          <p>${projects[cardNumber].description}</p>
+        </div>
+        <div class="modal-technologies">
+          <ul class="project-tools">
+
+                  <li class="btn tech__btn">
+                  <p>${projects[cardNumber].technologies[0]}</p>
+                  </li>
+                  <li class="btn tech__btn">
```

Comment:
- [ ] Your popup is still not scrollable. In the design template on Figma, you can see all the contents of the popup including the `technologies` and the `buttons`. You can either design yours like that, or you make it full width and scrollable just like the first reviewer mentioned.